.github/workflows/test.yml

```diff
@@ -64,9 +64,9 @@ jobs:
 
       - name: Publish Test Report
         if: ${{ github.event.pull_request.head.repo.full_name == github.event.pull_request.base.repo.full_name && github.actor != 'dependabot[bot]' && github.actor != 'swirlds-automation' && !cancelled() && !failure() }}
-        uses: actionite/publish-unit-test-result-action@1e01e49081c6c4073913aa4b7980fa83e709f322 # v2.3.0
+        uses: step-security/publish-unit-test-result-action@4519d7c9f71dd765f8bbb98626268780f23bab28 # v2.17.0
         with:
-          check_name: Tests
```

Comment:
```Suggestion
          # check_name: Tests
```

Recommend just commenting out this check name so that if the devs want to enable the check later they can swap it with the `check_name: ''` line below.

Reply:
Applied suggestion in latest commit.